src/s2_data/field_descriptors.py

```diff
@@ -0,0 +1,685 @@
+from dataclasses import dataclass
+from enum import IntFlag, IntEnum
+import typing
+from .field_types import *
+
+class CharacterUnlockFlags(IntFlag):
```

Comment:
Just personal preference but I tend to find shifts easier to read for correctness

```py
    Ana        = 1 << 0
    Margaret   = 1 << 1
    Colin      = 1 << 2
    Roffy      = 1 << 3
    Alto       = 1 << 4
    Liz        = 1 << 5
    Nekka      = 1 << 6
```
etc.